sui-agent/src/transactions/TransactionTool.ts

```diff
@@ -2,8 +2,6 @@ import { SuiClient, SuiHTTPTransport } from '@mysten/sui.js/client';
 import { TransactionBlock } from '@mysten/sui.js/transactions';
```

Comment:
Btw, I saw that `sui.js` is sort of deprecated, can we change to `@mysten/sui/transactions`instead ? And do this across the entire repo ?